anti-corruption-layer/src/main/java/com/iluwatar/corruption/system/legacy/LegacyOrder.java

```diff
@@ -0,0 +1,117 @@
+package com.iluwatar.corruption.system.legacy;
+
+import java.util.Objects;
+
+/**
+ * The class represents an order in the legacy system.
+ * The class is used by the legacy system to store the data.
+ */
+public class LegacyOrder {
```

Comment:
Use Lombok to get rid of boilerplate such as getters, setters, constructors, etc.